### ISSUE TITLE
issue

### ISSUE THREAD

**Issue by chandra1434** [luarocks/luarocks#1356]
### Please read the following before submitting:
- Please do NOT submit bug reports for questions. Ask questions in the [Gitter
  chat-room](https://gitter.im/luarocks/luarocks).
- Please do NOT submit duplicate bug reports. Look through the [issue
  tracker](https://github.com/luarocks/luarocks/issues) before submitting.

### Please fill out the following:
- **Platform:**
  - Windows/Linux/MacOS.

- **LuaRocks version:**
  - `luarocks --version`

- **Configuration file:**
  - Please try to produce with the default configuration.
  - If you cannot reproduce with the default configuration, please try to find
    the minimal configuration to reproduce.
  - Upload the config to a pastebin such as gist.github.com.

- **LuaRocks output from when the issue occurred:**
  - Use the `--verbose` flag.

- **Description:**
  - The steps you took in plain English to reproduce the problem.

**Comment by hishamhm** [comment]
Closing due to lack of issue-specific information. If you're having an issue, please open one with the information requested above.
